aztk/config.py

```diff
@@ -76,37 +75,17 @@ def load_secrets_config(self, path: str=constants.DEFAULT_SECRETS_PATH):
 
     def _merge_dict(self, secrets_config):
         # Ensure all necessary fields are provided
-        try:
-            self.batch_account_name = secrets_config['batch']['batchaccountname']
-        except KeyError:
-            raise error.AztkError(
-                "Please specify a batch account name in your .aztk/secrets.yaml file")
```

Comment:
are you validating anywhere now if the config is not valid?

Reply:
Yeah, validation is done in azure_api.py still. This was duplicating that so I removed it.